src/mono/mono/mini/mini-arm64.c

```diff
@@ -3352,6 +3403,28 @@ mono_arch_output_basic_block (MonoCompile *cfg, MonoBasicBlock *bb)
 		sreg2 = ins->sreg2;
 		imm = ins->inst_imm;
 
+		char handler_kind = opcode_simd_status[ins->opcode - OP_START];
+		if (handler_kind == OPCODE_SIMD)
+		{
```

Comment:
Coding style.

Reply:
`handler_kind` has been removed, since it only has to be checked once. Bracket spacing is fixed.